watchman/checks.py

```diff
@@ -66,13 +76,35 @@ def _check_email():
     return response
 
 
-def caches_status(request):
+def _check_storage():
+    try:
+        filename = 'django-watchman-{}.txt'.format(uuid.uuid4())
+        content = 'django-watchman test file'
+        path = default_storage.save(filename, ContentFile(content))
+        default_storage.size(path)
+        contents = default_storage.open(path).read()
```

Comment:
For this one (and two lines up), why not compare against the known values as well?